tests/inspect/inspect.go

```diff
@@ -474,5 +476,24 @@ func main() {
 		fmt.Printf("Annotation %s=%s\n", globalFlags.PrintAppAnnotation, body)
 	}
 
+	if globalFlags.CheckMountNS {
+		appMountNS, err := os.Readlink("/proc/self/ns/mnt")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
+		}
+		s1MountNS, err := os.Readlink("/proc/1/ns/mnt")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
+		}
+		if appMountNS != s1MountNS {
+			fmt.Println("check-mountns: SUCCESS")
+		} else {
+			fmt.Println("check-mountns: FAIL")
+			os.Exit(1)
```

Comment:
Printing "different" and "identical" should be more self-documented than "success" and "fail".